app/controllers/state_file/questions/w2_controller.rb

```diff
@@ -1,113 +1,34 @@
 module StateFile
   module Questions
     class W2Controller < QuestionsController
-      include ReturnToReviewConcern
-      before_action :load_w2s
-      before_action :load_w2, only: [:edit, :update]
+      before_action :load_w2
 
-      def self.show?(intake)
-        invalid_df_w2s(intake).any?
+      def self.show?(_)
+        false
```

Comment:
can we just remove W2Controller from the `_navigation.rb` files which would let us not have to define this `show` method?

Reply:
i forgot to remove this!